cmd/cli/cluster.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/moira-alert/moira"
+	"github.com/moira-alert/moira/database/redis"
+)
+
+func renameAnyTagsSubscriptionsKeyForwards(database moira.Database) error {
+	switch d := database.(type) {
+	case *redis.DbConnector:
+		err := d.Client().Rename(d.Context(), "moira-any-tags-subscriptions", "{moira-tag-subscriptions}:moira-any-tags-subscriptions").Err()
```

Comment:
For correct reversing is very important to have the same strings for keys. I suggest saving each key like a string constant to protect against misprints.

Reply:
Done